tests/tools/test_convert_conf.py

```diff
@@ -0,0 +1,75 @@
+import dgl
+import unittest, pytest
+from scipy import sparse as spsp
+from dgl.distributed import partition_graph
+from convert_partition_conf import etype2canonical_etype, check_if_old_version
+from collections import Counter
+import json
+import tempfile
+import os
+
+def create_random_hetero(type_n, node_n):
+    num_nodes = {}
+    for i in range(1, type_n+1):
+        num_nodes[f'n{i}'] = node_n
+    c_etypes = []
+    count = 0
+    for i in range(1, type_n):
+        for j in range(i+1, type_n+1):
+            count += 1
+            c_etypes.append((f'n{i}', f'r{count}', f'n{j}'))
+    edges = {}
+    for etype in c_etypes:
+        src_ntype, _, dst_ntype = etype
+        arr = spsp.random(num_nodes[src_ntype], num_nodes[dst_ntype], density=0.001, format='coo',
+                          random_state=100)
+        edges[etype] = (arr.row, arr.col)
+    return dgl.heterograph(edges, num_nodes), [':'.join(c_etype) for c_etype in c_etypes]
+
+@unittest.skipIf(dgl.backend.backend_name == "tensorflow", reason="TF doesn't support some of operations in DistGraph")
```

Comment:
as tests under //tests/distributed and //tests/tools are only triggered in pytorch. it's ok to remove these `skipIf`.

Reply:
removed for fix